react-app/src/components/SearchForm/index.tsx

```diff
@@ -65,7 +65,7 @@ export const SearchForm: FC<{
         )}
         {!!searchText && (
           <XIcon
-            className="absolute cursor-pointer top-0 bottom-0 w-6 h-6 my-auto left-[28rem]"
+            className="absolute cursor-pointer top-0 bottom-0 w-6 h-6 my-auto left-[40rem]"
```

Comment:
this looks good on 700px now, but it isn't in the right place on like 900px. 
![Screenshot 2024-12-12 at 12 07 05 PM](https://github.com/user-attachments/assets/75366ecb-56ab-41bd-aa7a-6798bfb39d05)

Instead of using left with a position, can you do something that always positions the x to the right with just a little padding from the end of the box? Like something with flexbox & justify-content: end?

Reply:
Duly noted. Thanks for the insight. Will make the adjustments.